### ISSUE TITLE
Villages and Kingdoms

### ISSUE THREAD

**Issue by sj134107** [AlexModGuy/Ice-and-Fire-Suggestions#895]
This would add to the medieval and spice things up a bit, a rare village type, called a Kingdom. Consisting of a large village with a castle in the center, or at the back. Large castle, the village would be surrounded with walls, and Maybe develop over time. Like a tycoon, the NPCs would be merchants, guards, etc. There'd be a king and  queen in the castle. Etc. this would add more spice to the mod and maybe some story. Considering the lacking of civilization mods for 1.15, this idea poofed into mind.

**Comment by Noahleon000** [comment]
yeah and possibly  on rare occasions a dungeon structure can spawn in the kingdom with a random ice and fire mob trapped inside  such as a cyclops and on rare occasions a chained dragon under the keep that which u could kill or let free and  watch the kingdom go up in smoke
